lib/miniprofiler.js

```diff
@@ -227,6 +228,7 @@ function include(id) {
    popupShowTimeWithChildren = options.popupShowTimeWithChildren || popupShowTimeWithChildren;
    popupRenderPosition = options.popupRenderPosition || popupRenderPosition;
    storage = options.storage || storage;
+   resourcePath = `${options.resourcePath || ''}/${resourcePath}`;

```

Comment:
With token-based replacement, wouldn't this result in a double `//` before the last segment? e.g. `/prefix//mini-profiler-resources/`

Reply:
Good catch!  I am now stripping any trailing slashes from options.resourcePath